apps/internal/slog/logger_121_below.go

```diff
@@ -0,0 +1,29 @@
+//go:build !go1.21
+
+package slog
+
+import "context"
+
+type Level int
+
+type Logger struct{}
+
+const (
+	Debug Level = iota
```

Comment:
```suggestion
	LevelDebug Level = iota
```
etc. These symbols must be the same for all supported versions of Go